app/views/articles/index.html.erb

```diff
@@ -44,22 +44,23 @@
         <nav class="crayons-tabs hidden s:flex" aria-label="View posts by">
           <ul class="crayons-tabs__list">
             <li>
-              <a data-text="Feed" href="<%= list_path %>/" class="crayons-tabs__item <%= "crayons-tabs__item--current" if %w[week month year infinity latest].exclude?(params[:timeframe]) %>" aria-current="<%= "page" if %w[week month year infinity latest].exclude?(params[:timeframe]) %>">Feed</a>
+              <a data-text="Feed" href="<%= list_path %>/" class="crayons-tabs__item <%= "crayons-tabs__item--current" if %w[week month year infinity latest].exclude?(params[:timeframe]) %>"<%= %w[week month year infinity
+                                                                                                                                                                                                     latest].exclude?(params[:timeframe]) ? 'aria-current="page"'.html_safe : "" %>>Feed</a>
```

Comment:
I'm not sure if this added line break was intentional.

Reply:
no that's from husky linting